components/Carousel.js

```diff
@@ -16,6 +16,7 @@ const CarouselKeyboardNavigation = dynamic(
 );
 
 export default function Carousel({ slides }) {
+  slides.sort((a, b) => a.position - b.position);
```

Comment:
We should sort the slides in `/pages/index.js` (see `getStaticProps`) so we don't do it on each render.